src/callbacks_stage/positivity_ruedaramirez_gassner.jl

```diff
@@ -0,0 +1,223 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+@doc raw"""
+    PositivityPreservingLimiterRuedaRamirezGassner(semi::AbstractSemidiscretization;
+                                                   beta_rho = 0.1, beta_p = 0.1,
+                                                   near_zero_tol = 1e-9,
+                                                   max_iterations = 10,
+                                                   root_tol = 1e-13, damping = 0.8,
+                                                   use_density_init = true,
+                                                   surface_flux_fv = semi.solver.surface_integral.surface_flux,
+                                                   volume_flux_fv = semi.solver.volume_integral.volume_flux_fv,
+                                                   alpha_max = semi.solver.volume_integral.indicator.alpha_max + 0.1)
```

Comment:
Is there a reason why you increased the number further?

Reply:
Right, so this is mentioned in the docstring. Adding 0.1 gives the limiter more room to act, i.e., have an effect for cells which have already max FV contribution according to the HG SC approach.